openlr/openlr_match_quality/openlr_assessment_tool/mainwindow.cpp

```diff
@@ -19,15 +26,21 @@
 
 #include <cerrno>
 #include <cstring>
+#include <vector>
 
 namespace
 {
 class TrafficDrawerDelegate : public TrafficDrawerDelegateBase
 {
+  static constexpr char const * kEncodedLineId = "encodedPath";
```

Comment:
Sorry, no. Linker doesn't like this. And I don't want to add definitions for them.